src/main/java/edu/hawaii/its/api/service/OotbGroupingPropertiesService.java

```diff
@@ -130,56 +141,159 @@ public GetGroupsResults getGroupsResults() {
         return getGroupsResults;
     }
 
-    /* Updating Member(s) */
+    public Boolean updateActiveUserProfile(List<String> authorities, String uid, String uhUuid, String name, String givenName) {
+        updateActiveUserProfileInGroupingsOotbs(authorities, uid, uhUuid, name, givenName);
+        updateActiveUserProfileInGetGroupResultsBean(authorities, uid, uhUuid, name, givenName);
+        updateActiveUserProfileInHasMembersResultsBean(authorities, uid, uhUuid, name, givenName);
+        addActiveUserProfileToSubjectListWithoutDuplicate(authorities, uid, uhUuid, name, givenName);
+        addActiveUserProfileToMembersListWithoutDuplicate(authorities, uid, uhUuid, name, givenName);
+        return true;
+    }
```

Comment:
I think it might be better to just put everything in this one function and use one or more helper functions instead of splitting it into 5 functions. There is a lot of repeated code for creating a WsSubject.